.gitignore

```diff
@@ -255,6 +255,9 @@ paket-files/
 .DS_Store
 .DS_Store?
 
+# Visual Studio code
+.vscode/
```

Comment:
https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/.vscode/settings.json

I like having this in the repo because it we can use to setup things like linting, etc in the repo.

Reply:
Ok cool I'll remove this from .gitignore